pages/browse/index.js

```diff
@@ -50,7 +50,7 @@ const Browse = () => {
           services.map(service => (
             <Item
               key={service.id}
-              item={service}
+              item={service || {}}
```

Comment:
```suggestion
              item={service}
```

Reply:
we're using the `ItemLoading` component now when we don't have services, so we don't need a fallback here.